### ISSUE TITLE
Typo Issue first Steps page?

### ISSUE THREAD

**Issue by IsikAkgun** [TYPO3-Documentation/TYPO3CMS-Tutorial-Typoscript45Minutes#42]
While reading the TS first Steps site the text says: 
```
# Change the copied object.
# The change has no effect on mypage.10; it still returns "Holla".
test.value = Hello world
```
https://docs.typo3.org/typo3cms/TyposcriptIn45MinutesTutorial/TypoScriptOverview/FirstSteps/Index.html

But shouldn't there stand 
`# Change the original Object `
instead?

The Tutorial set "test" first, then copied "test" into "mypage.10" and then changed "test.value".

**Comment by sypets** [comment]
Ok. We will check this out. Thanks for pointing it out. 

Your issue is a little unreadable because GitHub (markdown) interprets the `#` in your sample code as headers. In the future, please use code formatting for code snippets (click on `<>` in the panel above the text window on GitHub). I will fix this for you. 

If you need help right now, here are some resources for getting support: https://typo3.org/help.

**Comment by sypets** [comment]
@IsikAkgun Additionally, it is always really helpful for us to add a link to the page you are referring to. I added that to your text.

**Comment by sypets** [comment]
About your original question: Yes, I do believe you are right. Indeed, that is confusing.

If you wish, you can create a pull request to make the change yourself. Just click on "Edit me on GitHub" in top right of the page or see [CONTRIBUTING.md](https://github.com/TYPO3-Documentation/TYPO3CMS-Tutorial-Typoscript45Minutes/blob/latest/CONTRIBUTING.md) in this repo.

**Comment by IsikAkgun** [comment]
Alright. Thank you for fixing my text above. 
I'm doing the pull request now.

**Comment by sypets** [comment]
Was resolved with #43
